### PR TITLE
Added missing models

### PR DESCRIPTION
**Pull request by jelkosz**
Without this models the filtering of events on the events screen does not work.

Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1651471

### CODE REVIEW

File: frontend/public/models/index.ts [hunk 1]
```diff
@@ -814,6 +814,45 @@ export const VirtualMachineInstancePresetModel: K8sKind = {
   id: 'virtualmachineinstancepreset',
 };
 
+export const DataVolume: K8sKind = {
```
Comment (mareklibra) [inline frontend/public/models/index.ts]: This should go to kubevirt models `frontend/public/kubevirt/models/vm.ts`.
Once moved, we should merge this after https://github.com/kubevirt/web-ui/pull/120

Comment (jelkosz) [reply frontend/public/models/index.ts]: done

File: frontend/public/kubevirt/models/vm.ts [hunk 2]
```diff
@@ -40,6 +40,32 @@ export const VirtualMachineInstancePresetModel: K8sKind = {
   id: 'virtualmachineinstancepreset',
 };
 
+export const VirtualMachineInstanceReplicaSet: K8sKind = {
+  label: 'Virtual Machine Instance Replica Set',
+  labelPlural: 'Virtual Machine Replica Set',
```
Comment (rawagner) [inline frontend/public/kubevirt/models/vm.ts]: should be Virtual Machine Instance Replica Sets

Comment (jelkosz) [reply frontend/public/kubevirt/models/vm.ts]: done

**Pull request merged**